src/account/AccountUtils.ts

```diff
@@ -0,0 +1,216 @@
+import { Deserializer, Serializer } from "../bcs";
+import { AnyPublicKeyVariant, HexInput, SigningScheme } from "../types";
+import { MultiKeyAccount } from "./MultiKeyAccount";
+import { Account } from "./Account";
+import { Ed25519Account } from "./Ed25519Account";
+import { isSingleKeySigner, SingleKeyAccount, SingleKeySignerOrLegacyEd25519Account } from "./SingleKeyAccount";
+import { KeylessAccount } from "./KeylessAccount";
+import { FederatedKeylessAccount } from "./FederatedKeylessAccount";
+import { AbstractKeylessAccount } from "./AbstractKeylessAccount";
+import {
+  AccountAddress,
+  Ed25519PrivateKey,
+  getIssAudAndUidVal,
+  Hex,
+  MultiKey,
+  Secp256k1PrivateKey,
+  ZeroKnowledgeSig,
+} from "../core";
+import { deserializeSchemeAndAddress } from "./utils";
+import { EphemeralKeyPair } from "./EphemeralKeyPair";
+
+/**
+ * Utility functions for working with accounts.
+ */
+export class AccountUtils {
```

Comment:
I don't think we need a static class to namespace these functions.
How about either exporting functions directly e.g.

```typescript
export function accountToBytes(account: Account): Uint8Array { ... }
export function accountFromHexInput(input: HexInput): Account { ... }
export function serializeAccount(serializer: Serializer, account: Account) { ... }
export function deserializeAccount(deserializer: Deserializer): Account { ... }
```

or alternatively, a namespace

```typescript
export namespace AccountUtils {
  export function ...
}
```

Reply:
Ah didn't know namespace was a thing